### DISCUSSION TITLE
2 persons on generated image almost all the time

### DISCUSSION THREAD

**Discussion by ykurilov** [TheLastBen/fast-stable-diffusion#557] [Q&A]
Hi!

I've just created a model with my photos using 4000 steps and 30% text encoder. 33 instance photos. 

But for some strange reasons on most of my images it creates 2 copies of me. Even on square ones. 

And I have already created few dozens of models. And didn't have such problems. What could it be?

**Comment by TheLastBen** [comment]
hi, 
must be the resolution too high, tick the box "highres.fix" in A1111 webui

**Reply by ykurilov** [reply]
The resolution wasn't high. Just as usual. Anyway I didn't like that model and created another. Don't have problems with new ones.

But I still cannot find a good balance between flexibility and likeness of model.

It has good likeness and not good enough flexibility. 

Tested with 3500 steps text encoder 30 and text encoder 50.

My other model build on another colab still better. But I feel some potential here too.

**Reply by TheLastBen** [reply]
Set the textenc to 10% and the steps to 6000, and make sure to use non-existing word in the images name

**Reply by ykurilov** [reply]
Ok! Thank you!

**Reply by nawnie** [reply]
did this fix the issue? i find i have this issue if my sources images contain a second person
